cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/RecordWriter.java

```diff
@@ -108,12 +109,23 @@ private String getStreamId(TaskContext taskContext)
     public List<StreamResult> write(Iterator<Tuple2<DecoratedKey, Object[]>> sourceIterator)
     {
         TaskContext taskContext = taskContextSupplier.get();
+        LOGGER.info("[{}]: Processing bulk writer partition", taskContext.partitionId());
```

Comment:
debug level?

Reply:
This is an existing log line that was moved up to the beginning of the method.